Source/Blazorise/Base/BaseInputComponent.razor.cs

```diff
@@ -125,6 +149,20 @@ protected virtual void ReleaseResources()
         }
     }
 
+    /// <summary>
+    /// Initializes parameters that are not going to be changed again.
+    /// </summary>
+    protected void IntializeParameters()
+    {
+        if ( hasInitializedParameters )
+            return;
+
+        // we only need to generate field names on the server side
+        shouldGenerateFieldNames = !OperatingSystem.IsBrowser();
+
+        hasInitializedParameters = true;
```

Comment:
Might prefer to move this up, just to be sure there's no way this code can execute twice? Although I think it probably never will.

Reply:
I doubt. The logic is practically the same as in .NET project. If they think it will not be executed twice I guess it is fine.

Reply:
Don't forget that Microsoft employees are still humans, and they can also make mistakes. Anyway I also doubt, I just took a look inside IsBrowser and its returning super immediately, so yes.
The "correct way" for this case, could be as stated.